src/modules/grubcfg/main.py

```diff
@@ -51,20 +55,26 @@ def modify_grub_default(partitions, root_mount_point, distributor):
             if partition["fs"] == "linuxswap":
                 swap_uuid = partition["uuid"]
 
-            if partition["fs"] == "linuxswap" and "luksMapperName" in partition:
+            if partition[
+                    "fs"] == "linuxswap" and "luksMapperName" in partition:
                 swap_outer_uuid = partition["luksUuid"]
```

Comment:
Please don't split it like this; use

```
if (partition["fs"] == "linuxswap"
    and "luksMapperName" in partition):
```

(possibly with "and" at the end of the line -- I'm not sure what PEP8 says on that front). In any case the way you've line-broken here interferes with reading.